packages/xo-web/src/xo-app/backup-ng/new/smart-backup.js

```diff
@@ -4,16 +4,17 @@ import defined, { get } from '@xen-orchestra/defined'
 import Icon from 'icon'
 import PropTypes from 'prop-types'
 import React from 'react'
-import SmartBackupPreview, {
-  constructSmartPattern,
-  destructSmartPattern,
-} from 'smart-backup'
 import Tooltip from 'tooltip'
 import { connectStore, resolveIds } from 'utils'
 import { createGetObjectsOfType } from 'selectors'
+import { flatten } from 'lodash'
 import { injectState, provideState } from 'reaclette'
 import { Select } from 'form'
 import { SelectPool, SelectTag } from 'select-objects'
+import SmartBackupPreview, {
+  constructSmartPattern,
+  destructSmartPattern,
+} from 'smart-backup'
```

Comment:
This is not sorted the same way as this https://github.com/vatesfr/xen-orchestra/pull/4191/files#diff-9c044ab8a42ed6576ea927a64c1ec3ebR45